functions/utils/models.py

```diff
@@ -83,11 +83,11 @@ class Meta:
     nestComponent = UnicodeAttribute(range_key=True, attr_name='nestComponent')
     title = UnicodeAttribute(attr_name='title')
     owner = UnicodeAttribute(attr_name='owner', null=True)
-    description = UnicodeAttribute(attr_name='description', null=True)
+    description = UnicodeAttribute(attr_name='description', null=True, default='')
```

Comment:
How does this work for existing stories where the description is already null?  And ideally the `null` flag should not be present if there is a default

Reply:
I'm asking this because it seems like you added the default so that you will get empty string, but for existing records with null as the value will it return back `null` or `''`?  What happens if you keep the default but remove the `null=True` flag?

Reply:
I just tested something like this.  It's fine for backwards compatibility, just remove the `null=True` flag for both of these changes.